src/main/java/io/epirus/console/config/CliConfig.java

```diff
@@ -12,24 +12,26 @@
  */
 package io.epirus.console.config;
 
-import java.io.*;
+import java.io.File;
+import java.io.IOException;
 import java.nio.charset.Charset;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.UUID;
 
 import com.google.gson.Gson;
+import com.google.gson.annotations.Expose;
 
 import org.web3j.utils.Version;
 
 public class CliConfig {
-    private static final Path EPIRUS_CONFIG_PATH =
+    private static final Path DEFAULT_EPIRUS_CONFIG_PATH =
             Paths.get(System.getProperty("user.home"), ".epirus", ".config");
-    private static final String defaultServicesUrl = "https://auth.epirus.io";
+    private static final String servicesURL = "https://auth.epirus.io";
```

Comment:
This URL is also defined earlier in the account manager. We shouldn't redefine it, better to have one location for it.

Reply:
Yes, you are right. This is my fault as I misunderstood the purpose of this. The address is needed in the updater in order to get the latest CLI version. It is completely unrelated to the deploy command and I have changed it to its initial value:https://internal.services.web3labs.com